python/tk_multi_loader/delegate_publish_list.py

```diff
@@ -73,36 +74,12 @@ def _format_folder(self, model_index, widget):
         
         :param model_index: Model index to process
         :param widget: widget to adjust
-        """        
-        sg_data = shotgun_model.get_sg_data(model_index)
-        field_data = shotgun_model.get_sanitized_data(model_index, 
-                                                      shotgun_model.ShotgunModel.SG_ASSOCIATED_FIELD_ROLE)
+        """
 
-        # examples of data. In the examples below, when we say "node", this 
```

Comment:
Commenting the details of `sg_data` and `field_data` was moved to `model_item_data.get_item_data(item)` in order to maintain these comments in one location and avoid the current discrepancies here and there.